src/react-ruler/Ruler.tsx

```diff
@@ -288,13 +286,17 @@ export default class Ruler extends React.PureComponent<RulerProps> implements Ru
     context.restore();
   }
 
+  private _getTheme(theme: 'dark' | 'light'): ThemeInterface {
```

Comment:
I think theme comes from props
sp no need to use function arguments
use this.props instead and you will be able to convert method to the `get` like I wrote before